ngrok/README.md

```diff
@@ -20,18 +20,31 @@ Every service with a link or port-forward will have a new button labelled 'ngrok
 Clicking the button will create a new tunnel. You can see all the currently
 open tunnels at http://localhost:4040/.
 
+## Flags
+
+`--auth=user:password`: Adds a basic auth prompt to all tunnels.
+
 ## Examples
 
+Default behavior:
+
 ```python
 v1alpha1.extension_repo(name='default', url='https://github.com/tilt-dev/tilt-extensions')
 v1alpha1.extension(name='ngrok:config', repo_name='default', repo_path='ngrok')
 ```
 
-## Future Work
+With auth:
 
-### Authentication
+```python
+v1alpha1.extension_repo(name='default', url='https://github.com/tilt-dev/tilt-extensions')
+v1alpha1.extension(
+  name='ngrok:config', 
+  repo_name='default', 
+  repo_path='ngrok', 
+  args=['--auth=user:password'])
```

Comment:
It might be good to:
1) make clear this doesn't want the literal string "user:password", e.g. "myuser:mypassword"
2) exemplify not committing the password to the tiltfile, e.g., `'--auth=user:%s' % (read_file(os.path.join(config.main_dir, ".ngrok-password")))`